.env.local.example

```diff
@@ -43,6 +43,7 @@ WEB_CONTENT_FUNCTION_SECRET=SECRET
 # Resned email api key
 RESEND_EMAIL_API_KEY=KEY
 RESEND_EMAIL_RECEIVER=EMAIL
+RESEND_EMAIL_RECEIVER2=EMAIL
```

Comment:
@1orZero Sorry why do we have two Resend receiver?

Reply:
Its for sending error report to my email and yours. Maybe i should just harder it in the codebase right?

Reply:
ah i see, yea it's alright in this case actually